addons/auth_password_policy_signup/static/src/public/scss/signup_policy.scss

```diff
@@ -3,4 +3,8 @@
   meter.o_password_meter {
     bottom: calc(#{$input-height} / 2 - 7px);
   }
+
+    #password {
+        padding-right: 6rem;
```

Comment:
- Seems random, should it be based on the meter width?
- Maybe this deserves its own commit, maybe even in 18.0 at least? :shrug:

Reply:
I made a 18.0 PR with the suggested change https://github.com/odoo/odoo/pull/200522
Since I changed many lines, I seized the opportunity to reindent the file as well, it was using space: 2 instead of 4